ui/src/components/Menu.svelte

```diff
@@ -67,9 +73,9 @@
 
 </script>
 
-<div class="w-screen h-screen flex flex-row items-center justify-end" transition:fly={{ x: 400 }}>
+<div class="w-screen h-screen {isRightMargin ? 'mr-[5%]' : 'ml-[-70%]'} flex flex-row items-center justify-end" transition:fly={{ x: 400 }}>
```

Comment:
the justify-end is doing so the menu is on the right side, so justify-start would make it be on left side

How it should be
```
class="w-screen h-screen flex flex-row items-center { isRightMargin ? 'justify-end' : 'justify-start' }"
```